### DISCUSSION TITLE
jupyterlab-myst and internationalization

### DISCUSSION THREAD

**Discussion by nthiery** [executablebooks/meta#912] [Q&A]
I am starting to use jupyterlab-myst for my myst-markdown jupyter notebooks.
That's really nice. Admonitions, finally!!!

Questions: my class material is in French. So I would like
```
:::{warning}
...
:::
```
to render in JupyterLab with title `Avertissement` rather than `Warning`, as is the case
in the static web site produced by JupyterBook.

I haven't seen this mentioned in the doc. Is it already available? Otherwise, should I
make a feature request?

Cheers,
                       Nicolas

**Comment by rowanc1** [comment]
Would love an issue on the project! I am currently working on some significant improvements to the library, and hoping that we can think about this, at least in part, as part of that work!  More about some of the current progress is here:
https://github.com/executablebooks/jupyterlab-myst/issues/57

![image](https://user-images.githubusercontent.com/913249/211687343-fe874c51-a6fe-485e-95af-d0e2d4ec71a7.png)

An issue would be much appreciated, where we can talk about how to configure this (currently the extension doesn't look to the sphinx/jupyterbook config, and so will have to ask for some pointers from folks on how best to do this!).

**Comment by nthiery** [comment]
Thanks @rowanc1 for the quick feedback! Issue here: https://github.com/executablebooks/jupyterlab-myst/issues/58

**Reply by rowanc1** [reply]
Awesome, thank you for kicking this off. Exciting to see the discussion already!
